### PR TITLE
Sphinx breathe rtd documentation

### LINKED ISSUE #16: Sphinx-Breathe-RTD documentation

### PR DESCRIPTION
**Pull request by dokempf**
This fixes #16

### CODE REVIEW

**Comment by dokempf** [comment]
The general Sphinx->breathe->RTD setup is ready for review, I am only looking into minor test issues currently.

File: {{cookiecutter.project_slug}}/README.md [hunk 1]
```diff
@@ -27,9 +28,12 @@ cd build
 cmake -DCMAKE_BUILD_TYPE=Release ..
 cmake --build .
 ```
-{% if cookiecutter.doxygen == "Yes" %}
-# Documentation
 
+# Documentation
+{% if cookiecutter.readthedocs == "Yes" %}
+{{ cookiecutter.project_name }} provides a Sphinx-based documentation, that can
+be browsed [online at readthedocs.org](https://readthedocs.org).
```
Comment (lkeegan) [inline {{cookiecutter.project_slug}}/README.md]:
```suggestion
be browsed [online at readthedocs.org](https://{{ cookiecutter.project_slug }}.readthedocs.io).
```

**Comment by dokempf** [comment]
Now, also the testing stuff is ready to go in.

**Review by lkeegan — CHANGES_REQUESTED** [review]
I tried cookiecutter locally with this branch & following the TODO.

In the TODO we should explicitly say what the github repo name should be (just to check I used a different name, which works but some links are of course broken): https://github.com/lkeegan/test-cpp

Readthedocs build also failed because of this `main` vs `master` branch thing: https://readthedocs.org/projects/test-cpp/builds/12530324/

Otherwise looks very good!

**Comment by dokempf** [comment]
The `master` vs `main` thing with Readthedocs also tripped me over. They will change that: https://github.com/readthedocs/readthedocs.org/issues/7327 but it will take more time. Currently, it needs to be manually changed. I will add that to the TODO

File: {{cookiecutter.project_slug}}/README.md (hunk 1 above)
Comment (dokempf) [reply {{cookiecutter.project_slug}}/README.md]: I had already realized that as well, so your patch does not apply anymore, but the content is on the branch.

**Review by lkeegan — APPROVED** [review]

**Pull request merged**